pytorch_lightning/callbacks/quantization.py

```diff
@@ -174,7 +208,21 @@ def _check_feasible_fuse(self, model):
 
     def on_fit_start(self, trainer, pl_module):
         # QuantStub converts tensors from floating point to quantized
-        pl_module.quant = torch.quantization.QuantStub()
+        if self.max_num_inputs is None:
+            if pl_module.example_input_array is None:
+                num_inps = 1
+                rank_zero_warn(
+                    'Defaulting to singular QuantStub. Use `max_num_inputs` or `model.example_input_array` '
+                    'if your model takes multiple inputs', UserWarning
+                )
+            else:
+                input_sample = pl_module.example_input_array
+                if not isinstance(input_sample, (tuple, list)):
+                    input_sample = (input_sample, )
```

Comment:
Why is this necessary?

Reply:
I suppose it's not strictly necessary since `len(torch.Tensor) == len(tuple(t))`. I thought it might make that behavior more readable in light of the docstring changes, but can remove if you'd like